seed_data/management/commands/authorization_user_exam.py

```diff
@@ -0,0 +1,99 @@
+"""
+Authorize for exam of passed users
+"""
+import logging
+import datetime
+from django.core.management import BaseCommand
+
+from courses.models import Program
+from dashboard.api_edx_cache import CachedEdxUserData
+from dashboard.models import CachedEnrollment, ProgramEnrollment
+from dashboard.utils import MMTrack
+from exams.models import (
+    ExamProfile,
+    ExamAuthorization
+)
+from seed_data.utils import add_year
+
+
+log = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    Authorizations of exam of passed users
+    """
+    help = "Trigger exam authorizations when users already passed course"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--username",
+            dest="username",
+            default=None,
+            help="Username who will be authorized for exam if he passed a course(s).",
+            type=str
+        )
+        parser.add_argument(
+            "--program-id",
+            dest="program_id",
+            default=None,
+            help="Program id where we want to authorize users who passed course(s).",
+            type=int
+        )
+
+    def handle(self, *args, **options):
+        """
+        Trigger exam authorizations for user for a course if he has passed it.
+        """
+        program_id = options.get("program_id", None)
+        username = options.get("username", None)
+
+        programs = (
+            Program.objects.filter(id=program_id)
+            if program_id else Program.objects.filter(exam_series_code__isnull=False, live=True)
```

Comment:
That's fine, I agree on the centralized code, although I think this command itself should be in the `exams` app rather than `seed_data` since it doesn't have anything to do with `seed_data`.